api/sdk/sdk.go

```diff
@@ -182,3 +182,29 @@ func (sdk *SDK) Transfer(amount uint, from *Member, to *Member) (string, error)
 
 	return response.TraceID, nil
 }
+
+func (sdk *SDK) GetBalance(m *Member) (uint64, error) {
```

Comment:
Why uint64, and not uint?

Reply:
In fact I would prefer to use some sort of arbitrary precision numbers since we never told that balance is limited to 2^64 coins or any other number. But since currently we don't use arbitrary precision numbers for anything I chose an unsigned integer with largest precision.